lib/network.js

```diff
@@ -384,6 +385,51 @@ function _setDefaults(config) {
 }
 
 
+function _applyOverrides(config) {
+
+  function applyOverrideValue(value, fn) {
+    if (value) {
+      fn(value);
+    }
+  }
+
+  function applyEnvOverride(name, fn) {
+    applyOverrideValue(process.env[name], fn);
+  }
+
+
+  _.forEach(config.overrides, function(override) {
+
+    const flag = override.env && override.env.flag;
+    if (flag && process.env[flag]) {
+
+      applyEnvOverride(override.env.port, function(value) {
```

Comment:
given that this is specified in the same config file as the `edgemicro` object, what does providing a port override in a separate top level object and then copying it over into `edgemicro` provide?